src/Backends/WaylandBackend.cpp

```diff
@@ -2409,6 +2456,18 @@ namespace gamescope
         }
     }
 
+    wp_image_description_v1 *CWaylandBackend::CreateManualWindowsSCRGB()
+    {
+        // Attempt to manually create something approximating an scRGB image description:
+        // https://invent.kde.org/plasma/kwin/-/blob/master/src/wayland/colormanagement_v1.cpp#L94-107
+        // https://wayland.app/protocols/color-management-v1#wp_color_manager_v1:request:create_windows_scrgb
+        wp_image_description_creator_params_v1 *pParams = wp_color_manager_v1_create_parametric_creator( m_pWPColorManager );
+        wp_image_description_creator_params_v1_set_primaries_named( pParams, WP_COLOR_MANAGER_V1_PRIMARIES_SRGB );
+        wp_image_description_creator_params_v1_set_tf_named( pParams, WP_COLOR_MANAGER_V1_TRANSFER_FUNCTION_EXT_LINEAR );
+        wp_image_description_creator_params_v1_set_luminances( pParams, 0, 10000, 203 );
```

Comment:
This needs to check if the compositor supports parametric set_luminances request